package.json

```diff
@@ -56,7 +56,8 @@
   },
   "peerDependencies": {
     "react": "*",
-    "react-dom": "*"
+    "react-dom": "*",
+    "babel-polyfill": "*"
```

Comment:
I don't think we want to require a polyfill for such a minor issue. The change suggested in https://github.com/mui-org/material-ui/issues/15833#issuecomment-495793330 seems much more appropriate.

If react itself would require one I'd be fine with keeping the Symbol but they have a fallback for non-Symbol browsers as well.

Reply:
I made the changes and remove babel-polyfill